src/System.Web.OData/OData/Query/Expressions/PropertyContainer.cs

```diff
@@ -19,7 +19,7 @@ namespace System.Web.OData.Query.Expressions
     /// generic derived types that are used in the expressions that SelectExpandBinder generates.
     /// Also, Expression.Compile() could fail with stack overflow if expression is to deep and causes to mane levels of recursion. To avoid that we are tree-like property container.
```

Comment:
and causes "too" / we are "using" tree-like property...

Reply:
#Resolved